### PR TITLE
Sc-19916 : Minor fixes & Copy Updates

### PR DESCRIPTION
**Pull request by masskoder**
### Scope of changes

The following updates have been made :

 - Updates copy on the contact form
 - fix the pending registration issue
 - fix member status
 - GDS Cert Mgmt: Fix White Space

### Type of change

- [x] bug fix
- [ ] new feature
- [ ] documentation
- [x] other (describe)

### Acceptance criteria

https://www.awesomescreenshot.com/video/19988919?key=1f6cd4c157bc93b93aa70743702be09a

### Author checklist

- [x] I have manually tested the change and/or added automation in the form of unit tests or integration tests
- [ ]  I have updated the dependencies list
- [ ]  I have recompiled and included new protocol buffers to reflect changes I made
- [ ]  I have added new test fixtures as needed to support added tests
- [x]   Check this box if a reviewer can merge this pull request after approval (leave it unchecked if you want to do it yourself)
- [ ]  I have moved the associated Shortcut story to "Ready for Review"

### Reviewer(s) checklist

- [ ] Any new user-facing content that has been added for this PR has been QA'ed to ensure correct grammar, spelling, and understandability.

### CODE REVIEW

File: web/gds-user-ui/src/hooks/useUploadFile.tsx [hunk 1]
```diff
@@ -40,7 +40,7 @@ const useUploadFile = () => {
     reader.onload = async (ev: any) => {
       const data = JSON.parse(ev.target.result);
       try {
-        const validationData = await validationSchema[0].validate(data, { abortEarly: false });
+        const validationData = await allValidationSchema.validate(data);
```
Comment (masskoder) [inline web/gds-user-ui/src/hooks/useUploadFile.tsx:43]: The validation should be applied to all schemas, not just the basic form. The `allValidationSchema` encompasses all the form schemas.

File: web/gds-user-ui/src/components/OrganizationProfile/OrganizationalDetail.tsx [hunk 2]
```diff
@@ -5,44 +5,53 @@ import { getNationalIdentificationLabel } from 'constants/national-identificatio
...
 
   return (
     <Stack py={5} w="full">
       <Stack bg={'#E5EDF1'} h="55px" justifyItems={'center'} p={4}>
         <Stack mb={5}>
           <Heading fontSize={20}>
-            <Trans id="TRISA Organization Profile:">TRISA Organization Profile:</Trans>
+            <Trans>
+              {`
+                Your ${network === 'mainnet' ? 'Mainnet' : 'Testnet'} TRISA Organization Profile:
```
Comment (bbengfort) [inline web/gds-user-ui/src/components/OrganizationProfile/OrganizationalDetail.tsx:49]:
```suggestion
                Your ${network === 'mainnet' ? 'MainNet' : 'TestNet'} TRISA Organization Profile:
```

File: node_modules/.yarn-integrity
[the remark this answers is not in the document]
Comment (masskoder) [reply node_modules/.yarn-integrity:1]: Good catch. This file is generated when by mistake i run `yarn install` in the root. will remove it. thank you

**Review by bbengfort — APPROVED** [review]
:shipit:

**Pull request merged**